fish-rust/src/trace.rs

```diff
@@ -0,0 +1,75 @@
+use crate::{
+    common::{escape_string, EscapeFlags},
+    ffi::{self, parser_t, wcharz_t},
+    wchar::{self, wstr, L},
+    wchar_ffi::WCharToFFI,
+};
+use std::sync::atomic;
+
+#[cxx::bridge]
+mod ffi2 {
```

Comment:
Is there a reason this is `ffi2` and not just `ffi` (or `trace_ffi`)?

Reply:
I'm importing ffi above. Changing this to the more meaningful `trace_ffi`, thanks! 😄